website/content/api-docs/system/replication/replication-dr.mdx

```diff
@@ -438,13 +438,34 @@ $ curl \
 
 ## Update DR secondary's primary
 
-This endpoint changes a DR secondary cluster's assigned primary cluster using a
-secondary activation token. This does not wipe all data in the cluster.
+The update endpoint changes the primary cluster assigned to a DR
+secondary cluster. Changing the primary cluster assignment does not
+ wipe data in the secondary cluster.
 
 This endpoint requires a DR Operation Token to be provided as means of
 authorization. See the [DR Operation Token API
 docs](#generate-disaster-recovery-operation-token) for more information.
 
+There are two ways to update the primary assignment:
+
+1. **Use a secondary activation token with the `token` parameter**. An
+    activation token is required after a DR failover when updating a newly
+    demoted DR secondary cluster.
+
+1. **Use primary cluster addresses with the `update_primary_addrs` parameter**.
+    During the update, cluster addresses are pinged one at a time via gRPC.
+    The first cluster to respond successfully is assigned as the new primary
+    address. Updating with cluster addresses is not allowed after a DR
+    failover when updating a newly demoted DR secondary cluster. This was added
+    in Vault 1.13.3.
```

Comment:
```suggestion
    failover when updating a newly demoted DR secondary cluster.
```

Unless the fact that the functionality was introduced in 1.13 actually matters to someone using Vault 1.14, it's not useful to mention it here. In general, docs should focus on documenting the current functionality to reduce the amount of mental "noise" folks have to sift through. Readers can (and should) reference the older versions of the docs for information on previous behavior.